sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationTranslateOptions.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System.Collections.Generic;
+
+namespace Azure.AI.Translation.Text
+{
+    /// <summary> Client options for TextTranslationClient.Translate </summary>
+    public partial class TextTranslationTranslateOptions
+    {
+        /// <summary>
+        /// Specifies the language of the output text. The target language must be one of the supported languages included
+        /// in the translation scope. For example, use to=de to translate to German.
+        /// It&apos;s possible to translate to multiple languages simultaneously by repeating the parameter in the query string.
+        /// For example, use to=de and to=it to translate to German and Italian.
+        /// </summary>
+        public IEnumerable<string> TargetLanguages { get; }
+        /// <summary>
+        /// Array of the text to be translated.
+        /// </summary>
+        public IEnumerable<string> Content { get; }
+        /// <summary>
+        /// A client-generated GUID to uniquely identify the request.
+        /// </summary>
+        public string ClientTraceId { get; set; }
+        /// <summary>
+        /// Specifies the language of the input text. Find which languages are available to translate from by
+        /// looking up supported languages using the translation scope. If the from parameter isn&apos;t specified,
```

Comment:
A user can interpret the suggestion in this comment as "look up the answer online", rather than "call the GetLanguages method and pass 'translation' as the scope parameter". I would recommend making it clearer. This also applies to other properties in this class with a similar doc comment, such as `TargetLanguages`, `FromScript`, `ToScript`, etc..